### PR TITLE
Dashrews/ROX-13082 UUID test updates PR 3 of 4

### PR DESCRIPTION
**Pull request by dashrews78**
## Description

This PR focuses on the test constants and test changes needed to move the string constants in testing to be valid UUIDs.  This one builds upon the work done in https://github.com/stackrox/stackrox/pull/3681.

## Checklist
- [x] Investigated and inspected CI test results
- [x] Unit test and regression tests added
~- [ ] Evaluated and added CHANGELOG entry if required~
~- [ ] Determined and documented upgrade steps~
~- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs]~(https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

The work for this one is focused on updating tests to handle UUIDs.  So the changes are the tests.

### CODE REVIEW

File: central/deployment/datastoretest/datastore_sac_test.go [hunk 1]
```diff
@@ -220,7 +220,7 @@ func (s *deploymentDatastoreSACSuite) setupMultipleDeploymentReadTest() ([]strin
 	deploymentID2 := deployment2.GetId()
 	deployment3 := s.pushDeploymentToStore(testconsts.Cluster2, testconsts.NamespaceB)
 	deploymentID3 := deployment3.GetId()
-	deployment4 := s.pushDeploymentToStore(otherClusterID, otherNamespace)
+	deployment4 := s.pushDeploymentToStore(testconsts.Cluster3, otherNamespace)
```
Comment (rhybrillou) [inline central/deployment/datastoretest/datastore_sac_test.go:223]: Please remove the unused `otherClusterID` constant from the file as well.

File: central/notifiers/common_test.go [hunk 2]
```diff
@@ -198,7 +199,7 @@ func TestGetAnnotationValueReturnsDefaultIfNoStoreReturnsError(t *testing.T) {
 
 	alert := fixtures.GetAlert()
 
-	nsStore.EXPECT().SearchNamespaces(gomock.Any(), gomock.Any()).Return(nil, errors.New("testttt"))
+	nsStore.EXPECT().SearchNamespaces(gomock.Any(), gomock.Any()).Return(nil, errors.New(fixtureconsts.Cluster1))
```
Comment (rhybrillou) [inline central/notifiers/common_test.go:202]: Is the error text used in a UUID field in the flow being tested ?

**Review by rhybrillou — APPROVED** [review]

File: pkg/fixtures/node.go [hunk 3]
```diff
@@ -37,7 +38,7 @@ func GetNodeWithUniqueComponents() *storage.Node {
 
 func getNodeWithComponents(components []*storage.EmbeddedNodeScanComponent) *storage.Node {
 	return &storage.Node{
-		Id:   "id",
+		Id:   fixtureconsts.Node1,
```
[the remark this answers is not in the document]
Comment (dashrews78) [reply pkg/fixtures/node.go:41]: Since the column is nullable it is valid to not have it.

File: central/notifiers/common_test.go (hunk 2 above)
Comment (dashrews78) [reply central/notifiers/common_test.go:202]: this is just a mock in here.  I really didn't even need to change the line.  I do test some of that error testing of the UUID in PR 4/4

**Review by c-du — APPROVED** [review]

**Review by md2119 — APPROVED** [review]
lgtm

**Pull request merged**